examples/c++/ntt/example.cu

```diff
@@ -58,11 +59,16 @@ int validate_output(const unsigned ntt_size, const unsigned nof_ntts, E* element
   return nof_errors;
 }
 
+using FpMilliseconds = std::chrono::duration<float, std::chrono::milliseconds::period>;
+#define START_TIMER(timer) auto timer##_start = std::chrono::high_resolution_clock::now();
+#define END_TIMER(timer, msg) printf("%s: %.0f ms\n", msg, FpMilliseconds(std::chrono::high_resolution_clock::now() - timer##_start).count());
+
+
 int main(int argc, char* argv[])
 {
   std::cout << "Icicle Examples: Number Theoretical Transform (NTT)" << std::endl;
   std::cout << "Example parameters" << std::endl;
-  const unsigned log_ntt_size = 20;
+  const unsigned log_ntt_size = 26;
```

Comment:
@svpolonsky You can add the "examples" label [here](https://github.com/ingonyama-zk/icicle/blob/main/.github/workflows/examples.yml#L26) and it will run on a specific machine

Reply:
I'd rather keep things simple and use `log_ntt_size = 20`